main.rb

```diff
@@ -0,0 +1,52 @@
+def foo
+  a=b
+  return 1
+end
+def foo
+  a=b
+  return 1
```

Comment:
__[development][rubocop]__
* Useless assignment to variable - `a`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Redundant `return` detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-explicit-return)